tests/sentry/incidents/endpoints/test_organization_alert_rule_available_action_index.py

```diff
@@ -110,3 +111,24 @@ def test_no_feature(self):
         self.create_team(organization=self.organization, members=[self.user])
         resp = self.get_response(self.organization.slug)
         assert resp.status_code == 404
+
+    def test_sentry_apps(self):
+        sentry_app = self.create_sentry_app(
+            name="foo", organization=self.organization, is_alertable=True, verify_install=False
+        )
+        self.create_sentry_app_installation(
+            slug=sentry_app.slug, organization=self.organization, user=self.user
+        )
+
+        with self.feature(
+            ["organizations:incidents", "organizations:integrations-sentry-app-metric-alerts"]
+        ):
+            resp = self.get_valid_response(self.organization.slug)
+
+        assert resp.data == [
+            build_action_response(
```

Comment:
@mgaeta we should add more tests for `build_action_response` for the new changes since we are using `build_action_response` as the spec for another test.